src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -21,35 +20,53 @@ export interface ManifestSecretEnv {
   projectId?: string;
 }
 
-// Note: In this version of the API we used "serviceAccountEmail" to refer to
-// something that may not be an email (e.g. it might be myAccount@ to be project-relative)
-// In future revisions we should change this.
-type Base = Omit<backend.ServiceConfiguration, "secretEnvironmentVariables"> & {
-  serviceAccountEmail?: string | null;
-};
-type EventTrigger = backend.EventTrigger & {
-  serviceAccountEmail?: string | null;
+// Note: v1 schedule functions use *Duration instead of *Seconds
+// so this version of the API must allow these three retryConfig fields.
+type ScheduleTrigger = build.ScheduleTrigger & {
+  retryConfig?: {
+    maxRetryDuration?: string | null;
+    minBackoffDuration?: string | null;
+    maxBackoffDuration?: string | null;
+  } | null;
 };
 
-export type ManifestEndpoint = Base &
-  backend.Triggered &
-  Partial<backend.HttpsTriggered> &
-  Partial<backend.CallableTriggered> &
-  Partial<{ eventTrigger: EventTrigger }> &
```

Comment:
The above type `EventTrigger` allowed legacy users to specify `serviceAccountEmail`, but I'm not seeing where that went, maybe a merge issue?

Reply:
It looks like it's in the root of the `WireEndpoint`, any reason why it's not in event trigger?

Reply:
I _think_ the endpoint's service account and event trigger's service account can be different, looking at why I dropped it from the trigger now (could be a mistake)

Reply:
I..._think_ what I've written, somehow, works, for EventTrigger.serviceAccountEmail, but adding this modified type back into the file also works and makes it substantially more readable IMO, so I'm just gonna do that.